src/Downloader.php

```diff
@@ -110,8 +110,7 @@ public function getRecipes(array $operations): array
                 }
             }
 
-            // FIXME: getNames() can return n names
-            $name = str_replace('/', ',', $package->getNames()[0]);
+            $name = str_replace('/', ',', $package->getName());
```

Comment:
This look suspicious to me: there must be a reason why we didn't write this code originally.
What should we do with the extra names? When does it happen in the first place?
This should be investigated to me.

Reply:
I agree, but I try to find out why and I still do not have the answer. `getNames()` always return `getName()` first :
https://github.com/composer/composer/blob/master/src/Composer/Package/BasePackage.php#L95

Reply:
Works for me. The FIXME was more about what to do when we have more names. So, I would keep the FIXME and change the currently implementation as you did with `getName()`.